packages/components/src/range-control/styles/range-control-styles.ts

```diff
@@ -57,7 +57,10 @@ const wrapperMargin = ( { marks, __nextHasNoMarginBottom }: WrapperProps ) => {
 	return '';
 };
 
-export const Wrapper = styled.div< WrapperProps >`
+export const Wrapper = styled( 'div', {
+	shouldForwardProp: ( prop: string ) =>
+		! [ 'color', '__nextHasNoMarginBottom', 'marks' ].includes( prop ),
+} )< WrapperProps >`
```

Comment:
This change is not related to the fix, but rather an improvement that I applied while debugging the DOM rendered by `RangeControl`. I noticed that the value of the `color` prop was being applied as an attribute, which is not expected.